libs/wingc/src/jsify.rs

```diff
@@ -129,11 +137,14 @@ impl<'a> JSifier<'a> {
 				"const {} = process.env.WING_IS_TEST === \"true\";",
 				ENV_WING_IS_TEST
 			));
+		} else {
+			output.open(format!("module.exports = function({{ {} }}) {{", STDLIB));
+			output.line(format!("const std = {STDLIB}.{WINGSDK_STD_MODULE};"));
```

Comment:
I think this can be moved outside of the `if` blocks along with line 35

Reply:
Done